src/jaxoplanet/experimental/starry/pijk.py

```diff
@@ -0,0 +1,140 @@
+from collections import defaultdict
+from collections.abc import Mapping
+from typing import Any
+
+import equinox as eqx
+import jax
+import jax.numpy as jnp
+import numpy as np
+from jax.experimental.sparse import BCOO
+
+from jaxoplanet.types import Array
+
+
+class Pijk(eqx.Module):
+    """A class to represent and manipulate spherical harmonics in the polynomial basis.
+    Several indices are used throughout the class:
+    - Indices (i, j, k) represent the order of the polynomials of (x, y, z), for example
+    (1, 0, 2) represents x * z^2.
+    - Indices (l, m) represent the orders of the spherical harmonics.
+    - Index n represent the index of the polynomial in the flattened array.
+
+    Flattened array `to_dense` and `from_dense` follow th convention from Luger et al. (2019).
```

Comment:
It looks like this line is [tripping up the linter](https://results.pre-commit.ci/run/github/353677214/1707494294.T0l5dHFhQiCykEIjsAvshg) since it is too long.